### PR TITLE
Grossed out grammar fix

### PR DESCRIPTION
**Pull request by ghost**
e x t r e m e l y  d i s g u s t e d with this grammar

Fixes https://github.com/tgstation/tgstation/issues/29704

### CODE REVIEW

**Comment by PraiseRatvar** [comment]
also ignore that *scream commit, just go about your day and approve the changes

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Denied

File: code/modules/mob/living/carbon/human/examine.dm [hunk 1]
```diff
@@ -226,11 +226,11 @@
 			msg += "[t_He] [t_is] quite chubby.\n"
 	switch(disgust)
 		if(DISGUST_LEVEL_GROSS to DISGUST_LEVEL_VERYGROSS)
-			msg += "[t_He] looks a bit grossed out.\n"
+			msg += "[t_He] look a bit grossed out.\n"
```
Comment (PraiseRatvar) [inline code/modules/mob/living/carbon/human/examine.dm]: changed it to "would look"

**Comment by GunHog** [comment]
This is why "they" is problematic when used as a singular gender-neutral pronoun. A visible face (not obscured by a full helmet or gas mask) is what examine uses to decide whether to display a humanoid's gender. If you only displayed this information when the face is exposed, you might be able to avoid having to use "they" altogether. 

The logic here is that examiners would only be able to tell disgust level based on someone's facial expression. (This assumes that a head is present!) I am not sure if there is a situation where a humanoid with an exposed face might not have a confirmed binary gender. If this is not the case, please disregard this comment.

File: code/modules/mob/living/carbon/human/examine.dm [hunk 2]
```diff
@@ -226,11 +226,11 @@
 			msg += "[t_He] [t_is] quite chubby.\n"
 	switch(disgust)
 		if(DISGUST_LEVEL_GROSS to DISGUST_LEVEL_VERYGROSS)
-			msg += "[t_He] looks a bit grossed out.\n"
```
Comment (ChangelingRain) [inline code/modules/mob/living/carbon/human/examine.dm]: nope, wrong way, do `look[p_s()]` instead

**Review by Cyberboss — APPROVED** [review]

**Comment by Qustinnus** [comment]
i'm going to add an if check for if they're masked that prevents them from seeing this message. thanks for pointing that out.

**Pull request merged**